front/src/BottomBar/index.js

```diff
@@ -0,0 +1,70 @@
+import styled from "styled-components";
+import FontAwesomeIcon from "@fortawesome/react-fontawesome";
+import faPlay from "@fortawesome/fontawesome-free-solid/faPlay";
+import faStop from "@fortawesome/fontawesome-free-solid/faStop";
+import faCog from "@fortawesome/fontawesome-free-solid/faCog";
+import faThLarge from "@fortawesome/fontawesome-free-solid/faThLarge";
+import React from "react";
+import StandardButton from "./StandardButton";
+import PropTypes from "prop-types";
+import { buttonColor } from "../theme";
+
+const BottomMenuBar = props => {
+  return (
+    <div className={props.className}>
+      <BottomBarButton
+        onClick={() => props.page.onSend("start")}
+        enabled={true}
+      >
+        <FontAwesomeIcon icon={faPlay} />
+      </BottomBarButton>
+      {props.page.state.activePage === "refbox" && (
+        <BottomBarButton
+          onClick={() => props.page.setState({ activePage: "settings" })}
+          enabled={true}
+        >
+          <FontAwesomeIcon icon={faCog} />
+        </BottomBarButton>
+      )}
+      {props.page.state.activePage === "settings" && (
+        <BottomBarButton
+          onClick={() => props.page.setState({ activePage: "refbox" })}
+          enabled={true}
+        >
+          <FontAwesomeIcon icon={faThLarge} />
+        </BottomBarButton>
+      )}
+      <BottomBarButton onClick={() => props.page.onSend("stop")} enabled={true}>
+        <FontAwesomeIcon icon={faStop} color="red" />
+      </BottomBarButton>
+    </div>
+  );
+};
+BottomMenuBar.propTypes = {
+  page: PropTypes.object.isRequired
```

Comment:
add the shape, see `React.PropTypes.shape`

Reply:
I have no idea how this works right now, and as the intention is to change the props when testable, it seems redundant to me to put time in it while this blocks other tickets.

Reply:
see henks proposed changes.

Reply:
or maybe I'll do it anyway, we'll see next commit.